### ISSUE TITLE
Random positioning of unlock or select password popup

### ISSUE THREAD

**Issue by simonsimon006** [mozilla-lockwise/lockwise-android#1054]
## Steps to reproduce
Use Lockwise as default passwordmanager. Go to arbitrary website with z.B. Firefox Preview. Tap onto password input field.

### Expected behavior
The popup appears under or at least close to the selected input field.

### Actual behavior
Popup to unlock Lockwise will appear at a random position on the screen, somtimes under the keyboard or at the top of the window. The popup is functional though, you can use it normally.

### Device & build information
* Device: LG G8S Android 9
* Build version: 5214

### Notes
Attachments: Screenshot
![Capture+_2019-11-07-09-28-23](https://user-images.githubusercontent.com/1763926/68372429-1e3c6d00-0141-11ea-8fc9-418c1fc86d6c.png)

**Comment by abodea** [comment]
Hi @simonsimon006 thanks for this report! 
I tried on the latest version but the Lockwise floating menu was displayed above the keyboard on `www.golem.de`.
Can you please provide the exact steps or the exact website/section you access? 

Regards,
Andrei

**Comment by simonsimon006** [comment]
Yes. It happens when you visit golem.de with an enabled adblocker. They then ask you to either perform a series of tasks (ad-related) or you can login to your pay-account at the bottom. And there it happens. Sry, I should have put that in the first post.
